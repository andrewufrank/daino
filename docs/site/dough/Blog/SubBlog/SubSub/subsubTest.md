---
title: subsubTest z
author: AUF
version: publish
visibility: public
date: Jan. 4, 2022
keywords: notest
abstract: subsubTest A silly text not needing an abstract.
# pageTemplate: page3.yaml
---

another test with title z for testing sort order



  An example post sorted at last
