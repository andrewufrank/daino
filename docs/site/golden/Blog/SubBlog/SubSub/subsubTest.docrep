Docrep {meta1 = MetaPage {dyFn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/SubBlog/SubSub/subsubTest.md", dyLink = "Blog/SubBlog/SubSub/subsubTest.md", dyLang = DLenglish, dyTitle = "subsub z", dyAbstract = "A silly text not needing an abstract.", dyAuthor = "AUF", dyDate = Just "Jan. 4, 2022", dyKeywords = "notest", dyBibliography = Nothing, dyStyle = Nothing, dyReferences = Nothing, dyNoCite = Nothing, dyPublish = Nothing, dyIndexPage = False, dyIndexSort = Nothing, dyIndexEntry = IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/SubBlog/SubSub/subsubTest.md", link = "Blog/SubBlog/SubSub/subsubTest.md", title = "subsub z", abstract = "A silly text not needing an abstract.", author = "AUF", date = "Jan. 4, 2022", publish = Nothing, indexPage = False, dirEntries = [], fileEntries = []}}, pan1 = Pandoc (Meta {unMeta = fromList [("abstract",MetaInlines [Str "A",Space,Str "silly",Space,Str "text",Space,Str "not",Space,Str "needing",Space,Str "an",Space,Str "abstract."]),("author",MetaInlines [Str "AUF"]),("date",MetaInlines [Str "Jan.",Space,Str "4,",Space,Str "2022"]),("keywords",MetaInlines [Str "notest"]),("pageTemplate",MetaInlines [Str "page3.yaml"]),("title",MetaInlines [Str "subsub",Space,Str "z"])]}) [Para [Str "another",Space,Str "test",Space,Str "with",Space,Str "title",Space,Str "z",Space,Str "for",Space,Str "testing",Space,Str "sort",Space,Str "order"],Para [Str "An",Space,Str "example",Space,Str "post",Space,Str "sorted",Space,Str "at",Space,Str "last"]]}