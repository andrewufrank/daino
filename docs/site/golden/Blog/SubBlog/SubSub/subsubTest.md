---
title: subsub z
author: AUF
date: Jan. 4, 2022
keywords: notest
abstract: A silly text not needing an abstract.
pageTemplate: page3.yaml
---

another test with title z for testing sort order



  An example post sorted at last
