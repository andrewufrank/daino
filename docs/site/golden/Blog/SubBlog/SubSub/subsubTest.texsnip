TexSnip {snipyam = MetaPage {dyFn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/SubBlog/SubSub/subsubTest.md", dyLink = "Blog/SubBlog/SubSub/subsubTest.md", dyLang = DLenglish, dyTitle = "subsub z", dyAbstract = "A silly text not needing an abstract.", dyAuthor = "AUF", dyDate = Just "Jan. 4, 2022", dyKeywords = "notest", dyBibliography = Nothing, dyStyle = Nothing, dyReferences = Nothing, dyNoCite = Nothing, dyPublish = Nothing, dyIndexPage = False, dyIndexSort = Nothing, dyIndexEntry = IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/SubBlog/SubSub/subsubTest.md", link = "Blog/SubBlog/SubSub/subsubTest.md", title = "subsub z", abstract = "A silly text not needing an abstract.", author = "AUF", date = "Jan. 4, 2022", publish = Nothing, indexPage = False, dirEntries = [], fileEntries = []}}, unTexSnip = "another test with title z for testing sort order\n\nAn example post sorted at last"}