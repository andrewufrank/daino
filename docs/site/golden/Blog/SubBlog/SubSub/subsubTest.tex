\documentclass[a4paper,10pt]{scrbook}
\usepackage{fontspec}
\usepackage[ngerman]{babel}
\usepackage{graphicx}
\usepackage{makeidx}
\usepackage{natbib}
\makeindex
\usepackage[colorlinks]{hyperref}
\providecommand{\tightlist}{%
\setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}

\begin{document}

another test with title z for testing sort order

An example post sorted at last


\printindex
\end{document}
